documentation/config_file_logbackxml.markdown

```diff
@@ -40,20 +40,30 @@ You can also change the logging level for JRuby logging from its defaults of `er
 
 You can change the file to which Puppet Server writes its logs in the `appender` section named `F1`. By default, the location is set to `/var/log/puppetlabs/puppetserver/puppetserver.log`:
 
-~~~ xml
+``` xml
 ...
     <appender name="F1" class="ch.qos.logback.core.FileAppender">
         <file>/var/log/puppetlabs/puppetserver/puppetserver.log</file>
 ...
-~~~
+```
 
 To change this to `/var/log/puppetserver.log`, modify the contents of the `file` element:
 
-~~~ xml
+``` xml
         <file>/var/log/puppetserver.log</file>
-~~~
+```
 
-Note that the user account that owns the Puppet Server process must have write permissions to the destination path.
+The user account that owns the Puppet Server process must have write permissions to the destination path.
+
+#### `scan` and `scanPeriod`
+
+Logback supports noticing and reloading configuration changes without requiring a restart, a feature Logback calls **scanning**. To enable this, set the `scan` and `scanPeriod` attributes in the `<configuration>` element of `logback.xml`.  Scanning can potentially degrade logging performance and is not enabled by default.
+
+``` xml
+<configuration scan="true" scanPeriod="60 seconds" >
```

Comment:
looks like extra space here before closing >

Reply:
I'll remove it here and from the verbatim example in the JIRA ticket.